features/guests_added.feature

```diff
@@ -11,15 +11,46 @@ Feature: See if you are attending an event
       | Other event       | Look, it changes colors     | 22-Apr-3019 | 422 Treeside Way, Berkeley, CA 94704   | stubhub.com/liquid    | 0        |
     And I am on the events page
 
-  Scenario: I register for an event:
+
+
+  Scenario: I register for an event with no demographics:
+    When I follow "rsvp_1"
+    Then I should see "Gender"
+    And I fill out the form with the following attributes:
+      | firstname    | Test           |
+      | lastname     | Guest          |
+      | email        | test@guest.com |
+    Then I should see "You have successfully registered!"
+    And I should see "Test Guest"
+
+  Scenario: Delete a guest
     When I follow "rsvp_1"
+    Then I should see "Gender"
     And I fill out the form with the following attributes:
       | firstname    | Test           |
       | lastname     | Guest          |
       | email        | test@guest.com |
     Then I should see "You have successfully registered!"
     And I should see "Test Guest"
 
+    Given I am on the events page
+    When I follow "event_1"
+    Then I should see "Delete"
+    And I follow "Delete"
+    Then I should see "Guest was successfully removed."
+
+  Scenario: I register for an event with demographics:
+    When I follow "rsvp_1"
+    And I select "Undergrad" from "guest[occupation]"
```

Comment:
did not fix this.

Reply:
Fill out the form doesn't work with select boxes.  That's why I had to manually select each one.

Reply:
then create a step.